docs/.linkcheckerrc

```diff
@@ -1,3 +1,12 @@
 [filtering]
 ignore=
   ^https://stackoverflow\.com/
+
+ignorewarningsforurls=
+  # linkcheck always warns that it can't check markdown files,
+  # even though it does via the MarkdownCheck plugin
+  ^file:.*/README\.md$ ^url-content-type-unparseable
+
+
+[MarkdownCheck]
+filename_re=.*\.md$

```

Comment:
What does this do? Isn't it the default?

Reply:
It seems. I was probably trying some thing to help with... who knows... and added it and forgot to remove.